### PR TITLE
Fix for qt3 specific qt build

### PR DESCRIPTION
**Pull request by jgalarowicz**
A fix was added for setting LD_LIBRARY_PATH in the qt3 build, but if LD_LIBRARY_PATH is not set the qt build fails. So so check and set LD_LIBRARY_PATH if not set, update if it is set.

### CODE REVIEW

**Comment by jgalarowicz** [comment]
@adamjstewart Ok.  I will submit a change to just always set the LD_LIBRARY_PATH.   Seems like the travis checks failed, but not sure it is related to anything in this change?

**Comment by adamjstewart** [comment]
It's unrelated. See #3067 for an explanation of what happened. Several other PRs are failing their unit tests for the same reason, so don't worry, it's not your fault.

**Comment by jgalarowicz** [comment]
@adamjstewart @citibeth I will put back my original change that appended when LD_LIBRARY_PATH was set and sets LD_LIBRARY_PATH when it is not.

File: var/spack/repos/builtin/packages/qt/package.py [hunk 1]
```diff
@@ -226,8 +226,14 @@ def common_config_args(self):
 
     @when('@3')
     def configure(self):
-        # A user reported that this was necessary to link Qt3 on ubuntu
-        os.environ['LD_LIBRARY_PATH'] += os.pathsep + os.getcwd() + '/lib'
+        # A user reported that this was necessary to link Qt3 on ubuntu.
+	# However, if LD_LIBRARY_PATH is not set the qt build fails, check
+	# and set LD_LIBRARY_PATH if not set, update if it is set.
+	if not os.environ.get('LD_LIBRARY_PATH'):
```
Comment (citibeth) [inline var/spack/repos/builtin/packages/qt/package.py]: Can you swap the if and else clause?

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/qt/package.py]: The flake8 tests are crashing because you mixed tabs and spaces. Please only use spaces.

**Comment by jgalarowicz** [comment]
@adamjstewart @citibeth Hi Adam, Elizabeth - I fixed the source according to your suggestions.  Thanks.

**Pull request merged**